src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1020,7 +1020,11 @@ private void CheckRestrictedTypeReceiver(BoundExpression expression, CSharpCompi
 
             var receiver = ReplaceTypeOrValueReceiver(methodGroup.Receiver, !method.RequiresInstanceReceiver && !invokedAsExtensionMethod, diagnostics);
 
-            this.CoerceArguments(methodResult, analyzedArguments.Arguments, diagnostics, receiver?.Type, receiver?.GetRefKind());
+            var receiverRefKind = receiver?.GetRefKind();
+            uint receiverValEscapeScope = method.RequiresInstanceReceiver
+                ? receiverRefKind?.IsWritableReference() == true ? GetRefEscape(receiver, LocalScopeDepth) : GetValEscape(receiver, LocalScopeDepth)
+                : Binder.ExternalScope;
```

Comment:
Was this change observable in any test?

Reply:
Not that I currently have. I'm honestly not sure how to hit it.